grpc/proto/space.proto

```diff
@@ -167,9 +167,17 @@ service SpaceApi {
       body: "*"
     };
   }
+
+  rpc ListBuckets(ListBucketsRequest) returns (ListBucketsResponse) {
+    option (google.api.http) = {
+      get "/v1/buckets"
+    }
+  }
 }
 
-message ListDirectoriesRequest {}
+message ListDirectoriesRequest {
+  string bucket = 1;
```

Comment:
Okay so based on the discussion in Slack, I think we need `path` here as well. @dmerrill6 could you add it and regen it? Then I will use that as a base to fix the listdir(s) issues.